next.config.js

```diff
@@ -10,4 +12,4 @@ const nextConfig = {
   },
 };
 
-module.exports = nextConfig;
+module.exports = million.next(nextConfig)
```

Comment:
You did not integrate million.js, I don't see any optimiziation changes apart from adding million in config

Reply:
Is a draft.